src/GNDStk/gnds-json.hpp

```diff
@@ -1,6 +1,6 @@
 
 // -----------------------------------------------------------------------------
-// gnds::json
+// GNDStk::json
```

Comment:
This should be 
```c++
class JSON{
...}
```
And should be in the file `src/GNDStk/JSON.hpp`

Reply:
Per the NJOY naming convention, I've gone through and changed relevant jsons to JSONs, and xmls to XMLs. Note that changes weren't made in places that reference pugi xml (lower case) and nlohmann json (also lower case).